.github/fabricbot/updateAreaPodConfigs.js

```diff
@@ -526,12 +533,17 @@ let areaPodConfig = {
         "operator": "and",
         "operands": [
           {
-            "name": "isInProjectColumn",
-            "parameters": {
-            "projectName": `Area Pod: ${pod} - PRs`,
-            "columnName": "Needs Champion",
-            "isOrgProject": true
-            }
+            "operator": "not",
```

Comment:
Should we be allowing open PR's in the Done column? I agree that this rule makes sense for issues, but maybe not PRs.

Reply:
Perhaps for PRs move it to a new Column named something like "Moved to different Area" or something?

Reply:
Open PRs in the Done column would only be appropriate when the PR was moved to a different area. I don't think that happens _a lot_ with PRs though, so I propose we let them move into Done for now and if we see enough reason for it, we could introduce the other column.